### PR TITLE
Revise Getting Started section

### PR DESCRIPTION
**Pull request by rhcarvalho**
Specific changes to the Go SDK and general changes to all platforms.

### CODE REVIEW

File: src/collections/_documentation/error-reporting/getting-started-config/go.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+
+import (
+	"log"
+
+	"github.com/getsentry/sentry-go"
+)
 
 func main() {
-	sentry.Init(sentry.ClientOptions{
+	err := sentry.Init(sentry.ClientOptions{
```
Comment (rhcarvalho) [inline src/collections/_documentation/error-reporting/getting-started-config/go.md]: Adds error handling here. It is easy enough to ignore the error and end up with a misconfigured SDK without warning.

Although this makes the documentation more verbose, this is also where people will refer to. Better have the full picture when copy-pasting.

File: src/collections/_documentation/error-reporting/getting-started-verify/go.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+		log.Fatalf("sentry.Init: %s", err)
+	}
+	// Flush buffered events before the program terminates.
+	defer sentry.Flush(2 * time.Second)
 
-	sentry.CaptureException(errors.New("my error"))
-	// Since sentry emits events in the background we need to make sure
-	// they are sent before we shut down
-	sentry.Flush(time.Second * 5)
+	sentry.CaptureMessage("It works!")
```
Comment (rhcarvalho) [inline src/collections/_documentation/error-reporting/getting-started-verify/go.md]: For testing purposes, a message is simpler to setup than an error.

Prefer educating users to use `defer sentry.Flush(...)` instead of calling `Flush` after `CaptureException`.
People got the wrong impression that they need to call the two functions in sequence *all the time*.

File: src/collections/_documentation/error-reporting/quickstart.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ After you completed setting up a project in Sentry, you’ll be given a value wh
 
 ## Capturing your first event
 
-Once you have Sentry integrated into your project, you probably want to verify that everything is working as expected before deploying it, and what better way to do that than to break your application!
+Once you have Sentry integrated into your project, you probably want to verify that everything is working as expected before deploying it.
```
Comment (rhcarvalho) [inline src/collections/_documentation/error-reporting/quickstart.md]: Not all platforms suggest "breaking your application", therefore move this sentence to where it is appropriate, and leave it out elsewhere.

File: src/collections/_documentation/error-reporting/getting-started-verify/browser.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+What better way to send your first event than to break your application!
+
 One way to break your JavaScript application is to call an undefined function:
```
Comment (kamilogorek) [inline src/collections/_documentation/error-reporting/getting-started-verify/browser.md]: Maybe reword to "One way to do this (that?) is..." to not repeat "break your" twice?

Comment (rhcarvalho) [reply src/collections/_documentation/error-reporting/getting-started-verify/browser.md]: This repetition was already there for JS Browser, Cordova, JS Node, Electron. Now it just got obvious without rendering the docs :)


**CURRENT** https://docs.sentry.io/error-reporting/quickstart/?platform=browser#capturing-your-first-event
image


Not too bad, is it? I'll reword it since we're already touching this section.

File: src/collections/_documentation/error-reporting/getting-started-verify/browser.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
-One way to break your JavaScript application is to call an undefined function:
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/browser.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/browser.md [hunk 6]
```diff
@@ -1,4 +1,6 @@
...
+
+One way to do it is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/browser.md]:
```suggestion
Calling an undefined function will trigger a break:```

File: src/collections/_documentation/error-reporting/getting-started-verify/cordova.md [hunk 7]
```diff
@@ -1,4 +1,6 @@
-One way to break your Cordova application is to call an undefined function:
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/cordova.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/cordova.md [hunk 8]
```diff
@@ -1,4 +1,6 @@
...
+
+One way to do it is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/cordova.md]:
```suggestion
Calling an undefined function will trigger a break:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/csharp.md [hunk 9]
```diff
@@ -1,3 +1,5 @@
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/csharp.md]:
```suggestion
```

I'd remove this sentence because the sentence after communicates the same thing, but more specifically.

File: src/collections/_documentation/error-reporting/getting-started-verify/electron.md [hunk 10]
```diff
@@ -1,4 +1,6 @@
-One way to break your Electron application is to call an undefined function:
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/electron.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/electron.md [hunk 11]
```diff
@@ -1,4 +1,6 @@
...
+
+One way to do it is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/electron.md]:
```suggestion
Calling an undefined function will trigger a break:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/node.md [hunk 12]
```diff
@@ -1,4 +1,6 @@
-One way to break your JavaScript application is to call an undefined function:
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/node.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/node.md [hunk 13]
```diff
@@ -1,4 +1,6 @@
...
+
+One way to do it is to call an undefined function:
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/node.md]:
```suggestion
Calling an undefined function will trigger a break:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/php.md [hunk 14]
```diff
@@ -1,4 +1,6 @@
-You can trigger a PHP exception by throwing one in your application:
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/php.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/python.md [hunk 15]
```diff
@@ -1,4 +1,6 @@
-You can cause a Python error by inserting a divide by zero expression
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/python.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/rust.md [hunk 16]
```diff
@@ -1,3 +1,5 @@
+What better way to send your first event than to break your application!
```
Comment (MimiDumpling) [inline src/collections/_documentation/error-reporting/getting-started-verify/rust.md]:
```suggestion
One way to verify your setup is by intentionally sending an event that breaks your application. 
```

File: src/collections/_documentation/error-reporting/getting-started-verify/browser.md (hunk 6 above)
Comment (rhcarvalho) [reply src/collections/_documentation/error-reporting/getting-started-verify/browser.md]: "trigger a break" sounds really strange. "break" in JS is a keyword unrelated to exceptions.

How about "throw an exception", instead?

```suggestion
Calling an undefined function will throw an exception:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/csharp.md [hunk 17]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (rhcarvalho) [inline src/collections/_documentation/error-reporting/getting-started-verify/csharp.md]:
```suggestion
```

Gotta remove both lines (incl. empty line)... let's try out the brand new multi-line suggestions in GitHub :)

File: src/collections/_documentation/error-reporting/getting-started-verify/cordova.md (hunk 8 above)
Comment (rhcarvalho) [reply src/collections/_documentation/error-reporting/getting-started-verify/cordova.md]: (same as before)

```suggestion
Calling an undefined function will throw an exception:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/electron.md (hunk 11 above)
Comment (rhcarvalho) [reply src/collections/_documentation/error-reporting/getting-started-verify/electron.md]: (same as before)

```suggestion
Calling an undefined function will throw an exception:
```

File: src/collections/_documentation/error-reporting/getting-started-verify/node.md (hunk 13 above)
Comment (rhcarvalho) [reply src/collections/_documentation/error-reporting/getting-started-verify/node.md]: (same as before)

```suggestion
Calling an undefined function will throw an exception:
```

**Comment by rhcarvalho** [comment]
@MimiDumpling thank you very much for your suggestions! I applied them all with one modification:

```diff
- Calling an undefined function will trigger a break:
+ Calling an undefined function will throw an exception:
```

Because `break` is a keyword related to a different concept, and in this case "trigger a break" sounds weird.

---

@MimiDumpling would you like to have a final look?

File: src/collections/_documentation/error-reporting/getting-started-verify/browser.md (hunk 6 above)
Comment (MimiDumpling) [reply src/collections/_documentation/error-reporting/getting-started-verify/browser.md]: "throw an exception" is great!

**Comment by MimiDumpling** [comment]
> @MimiDumpling thank you very much for your suggestions! I applied them all with one modification:
> 
> ```diff
> - Calling an undefined function will trigger a break:
> + Calling an undefined function will throw an exception:
> ```
> 
> Because `break` is a keyword related to a different concept, and in this case "trigger a break" sounds weird.
> 
> @MimiDumpling would you like to have a final look?


Your suggestion to replace it with "throw an exception" is perfect. Thank you!
